k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -157,8 +158,8 @@
 
     private static final Set<Flag> SYNC_FLAGS = EnumSet.of(Flag.SEEN, Flag.FLAGGED, Flag.ANSWERED, Flag.FORWARDED);
 
-    private void suppressMessages(Account account, List<LocalMessage> messages) {
-        EmailProviderCache cache = EmailProviderCache.getCache(account.getUuid(), context);
+    private void suppressMessages(String accountUuid, List<LocalMessage> messages) {
```

Comment:
What motivated this change? It looks like all call sites just do the `Account.getUuid()` call now instead.